model/raw/src/main/scala/aqua/raw/ops/Call.scala

```diff
@@ -27,11 +28,30 @@ case class Call(args: List[ValueRaw], exportTo: List[Call.Export]) {
 object Call {
 
   // TODO docs
-  case class Export(name: String, `type`: Type) {
+  case class Export(name: String, `type`: Type, isExistingStream: Boolean = false) {
     def mapName(f: String => String): Export = copy(f(name))
 
     def toRaw: VarRaw = VarRaw(name, `type`)
 
     override def toString: String = s"$name:${`type`} <-"
   }
+  
+  extension (exportTo: List[Call.Export]) {
```

Comment:
Lets not defined those methods on a `List`? It is rather confusing I believe. better to use `map` on call site with methods defined on `Export`.